test_scripts/unit_tests/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y python procps git && apt-get clean
+ADD https://github.com/bats-core/bats-core/archive/refs/tags/v1.3.0.tar.gz /bats/
```

Comment:
I'd normally have build-args for the versions to allow you to pick up a new version without having to change the Dockerfile, you can also then add an env var for the build arg to detain it. Hadolint doesn't like it though so happy to let it fly for now! 👍

Reply:
We can look at this in a future release.  Added a todo for now